earth_enterprise/rpms/build.gradle

```diff
@@ -236,10 +242,21 @@ task openGeeCommonRpm(type: Rpm, dependsOn: openGeeSharedFiles) {
         'OpenSSL and ' +
         'Skia 5.1.2'
     type = BINARY
+    autoFindProvides = true
+    autoFindRequires = true
 
     requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
     conflicts('opengee-postgis', '2.0', GREATER | EQUAL)
-    requires('util-linux')
+
+    requireCommands(packageSharedCommands +
+        ['cat', 'cut', 'getent', 'grep', 'groupadd', 'groupdel'])
+    
+    /* Since some libraries (i.e., MrSid)  cannot be redistributed without a 
+       license and are not packaged as an RPM, we need to filter these out from the
+       automatic 'requires' dependency generation. */
+    
+    def filterCapabilities = ['libltidsdk.so()(64bit)']
+    autoFindRequiresFilter = { filterCapabilities.contains( 'libltidsdk.so()(64bit)') ? null : it }
```

Comment:
@tst-dsheremata: This condition will be always true, and you'll drop all requires.  I think you meant to write:

```Groovy
filterCapabilities.contains(it)
```